### PR TITLE
fix: fix url repo pkg

### PR DESCRIPTION
**Pull request by thedaviddias**
***Short description of what this resolves:***

Wrong URI for repo in package JSON

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "The Static Code Analysis Tool for your HTML",
   "repository": {
     "type": "git",
-    "url": "https://github.com/htmlhint/HTMLHint"
+    "url": "git+https://github.com//htmlhint/HTMLHint.git"
```
Comment (Shinigami92) [inline package.json:7]: I don't think we need the `//` before htmlhint
When I look into other repos like [here](https://github.com/vuetifyjs/vuetify/blob/dd9514be04036dd84953beac286cf255b77ece58/packages/vuetify/package.json#L13), there is no double `/`

Comment (waldyrious) [reply package.json:7]: Also, according to the [NPM docs](https://docs.npmjs.com/files/package.json#repository), we could probably use a simpler syntax:

    "repository": "npm/npm"

or

    "repository": "github:user/repo"

if we want to be more explicit.

Comment (Shinigami92) [reply package.json:7]: I don't know how well this works in terms of package.json: "click on link"

Comment (thedaviddias) [reply package.json]: Before Semantic Release uses that link, I prefer to keep it consistent with all repos.

**Pull request merged**